src/components/AvatarGroup/AvatarGroupCounterTooltipContent.tsx

```diff
@@ -54,30 +67,12 @@ const AvatarGroupCounterTooltipContent = ({
     className: displayAsGrid
       ? cx(styles.scrollableContainer, styles.tooltipContainer, styles.tooltipGridContainer, className)
       : cx(styles.scrollableContainer, styles.tooltipContainer, className),
-    direction: displayAsGrid ? Flex.directions.ROW : Flex.directions.COLUMN,
-    gap: displayAsGrid ? Flex.gaps.XS : Flex.gaps.SMALL,
+    direction: displayAsGrid ? FLEX_DIRECTIONS.ROW : FLEX_DIRECTIONS.COLUMN,
```

Comment:
It was not compiled with the Flex.directions.ROW syntax?

Reply:
It wasn't. I'll add TODOs so we can change it once Flex is converted to TS